### PR TITLE
[CRM457-1630] Rake task to set missing timestamps

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
Fix some of the missing data issues (still approx 12 left after these fixes

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1630)

### CODE REVIEW

File: lib/tasks/missing_version_timestamps.rake [hunk 1]
```diff
@@ -0,0 +1,133 @@
+desc "One off tasks to fix missing submission version timestamps"
+namespace :missing_version_timestamps do
+
+  desc "When the latest record missed the required timestamp"
+  task tranch1: :environment do
+    SubmissionVersion.where(created_at: nil)
```
Comment (patrick-laa) [inline lib/tasks/missing_version_timestamps.rake:6]: This is redundant, right? Unless it should be a guard clause?

**Review by jsugarman — COMMENTED** [review]
need to be wary of using `update` or `save[!]` as these rails methods update timestamps themselves.

I used `update_all`, `update_column` or raw sql to avoid this on my

**Review by patrick-laa — APPROVED** [review]
I think it'll work. And if it doesn't, I don't think it can go too far wrong.

File: lib/tasks/missing_version_timestamps.rake (hunk 1 above)
Comment (dwhenry) [reply lib/tasks/missing_version_timestamps.rake]: yes removed

**Review by jsugarman — APPROVED** [review]

**Pull request merged**